plugins/plugin-git/che-plugin-git-ext-git/src/main/java/org/eclipse/che/ide/ext/git/client/revert/RevertCommitPresenter.java

```diff
@@ -0,0 +1,211 @@
+/*
+ * Copyright (c) 2012-2017 Red Hat, Inc.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.che.ide.ext.git.client.revert;
+
+import static org.eclipse.che.api.git.shared.Constants.DEFAULT_PAGE_SIZE;
+import static org.eclipse.che.ide.api.notification.StatusNotification.DisplayMode.FLOAT_MODE;
+import static org.eclipse.che.ide.api.notification.StatusNotification.Status.FAIL;
+import static org.eclipse.che.ide.util.ExceptionUtils.getErrorCode;
+
+import com.google.inject.Inject;
+import com.google.inject.Singleton;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import org.eclipse.che.api.core.ErrorCodes;
+import org.eclipse.che.api.git.shared.RevertResult;
+import org.eclipse.che.api.git.shared.RevertResult.RevertStatus;
+import org.eclipse.che.api.git.shared.Revision;
+import org.eclipse.che.ide.api.app.AppContext;
+import org.eclipse.che.ide.api.dialogs.DialogFactory;
+import org.eclipse.che.ide.api.git.GitServiceClient;
+import org.eclipse.che.ide.api.notification.NotificationManager;
+import org.eclipse.che.ide.api.resources.Project;
+import org.eclipse.che.ide.ext.git.client.GitLocalizationConstant;
+import org.eclipse.che.ide.ext.git.client.outputconsole.GitOutputConsole;
+import org.eclipse.che.ide.ext.git.client.outputconsole.GitOutputConsoleFactory;
+import org.eclipse.che.ide.ext.git.client.revert.RevertCommitView.ActionDelegate;
+import org.eclipse.che.ide.processes.panel.ProcessesPanelPresenter;
+
+/**
+ * Presenter for reverting commits
+ *
+ * @author Dmitrii Bocharov (bdshadow)
+ */
+@Singleton
+public class RevertCommitPresenter implements ActionDelegate {
+  public static final String REVERT_COMMAND_NAME = "Git revert commit";
+  public static final String LOG_COMMAND_NAME = "Git log";
```

Comment:
Please make this constants private

Reply:
May be it would be better to move these constants to `GitLocalizationConstant`?

Reply:
For both remarks - i made it the same way it is done for the classes in this package: `RemotePresenter`, `RemoveFromIndexPresenter`, `ResetToCommitPresenter` etc...
Fix both anyway?

Reply:
That would be great.